action.yml

```diff
@@ -0,0 +1,31 @@
+name: Remote tflint Config
+description: Copies a tflint configuration file from a remote repo for local use
+author: ryanwholey
+inputs:
+  source-repo: 
+    required: true
+    description: The repository to copy a tflint configuration from
+  source-path:
```

Comment:
This is confusing, `path` should be a full path to the config. Or if you really need it accept dir/filename as inputs. But hopefully path is enough.